src/lib/core/backtrace.h

```diff
@@ -48,6 +48,9 @@ struct backtrace {
 
 struct fiber;
 
+void
+backtrace_init(void);
+
```

Comment:
What's this needed for?

Reply:
Sorry, this accidentally got into the diff.